.github/stale.yml

```diff
@@ -1,15 +1,15 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+daysUntilStale: 5
 
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 7
+daysUntilClose: 5
 
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - status:contributions-welcome 
-  - status:more-info-needed 
-  - status:duplicate
-  - type:enhancement
+  - contributions welcome
+  - feature request
+  - more info needed
```

Comment:
I think more info needed should be exempt, since this means the author did not provide sufficient information for troubleshooting. If it continues to be stale/ignored, it makes sense to close.